packages/wallets/src/components/WalletsCarouselHeader/WalletsCarouselHeader.tsx

```diff
@@ -39,6 +41,7 @@ const WalletsCarouselHeader: React.FC<TProps> = ({ balance, currency, isDemo })
             <IconButton
                 color='transparent'
                 data-testid='dt_wallets_carousel_header_button'
+                disabled={hidden}
```

Comment:
minor: how does "hidden" translates to "disabled"?

Reply:
This is to ensure the button is disabled when WalletsCarouselHeader is hidden as it's the only interactable part of the component